Feitos os exercícios: 1564, 1589, 1759 (#55)
petbccufscar/URI: Iniciante/1759/README.md

> # Problema 1759:

Reviewer: ```suggestion
# Problema:
```
* Importante seguir o guia de padronização

Papai Noel está brincando com seus duendes para entretê-los durante a véspera do Natal. A brincadeira consiste nos elfos escreverem números em pedaços de papel e colocarem no gorro do Papai Noel. Após todos terminarem de colocar os números Noel sorteia um papel e aquele número representa quantos "Ho" o Noel deve falar.

Seu trabalho é ajudar o Papai Noel montando um problema que mostre todos os "Ho" que ele deve falar dado o número sorteado.

###### Problema completo: https://www.urionlinejudge.com.br/judge/pt/problems/view/1759

# Resolução

> A ideia do exercício é usar uma variável de entrada para limitar uma estrutura de repetição e dentro da mesma printar `N` vezes **Ho Ho.. Ho!**, lembrando que o último "Ho" tem que ser seguido do "!" ficando "Ho!".
> Para isso iniciamos declarando as variáveis que serão utilizadas:

Reviewer: ```suggestion
A resolução do exercício consiste em utilizar uma variável para armazenar a entrada e limitar a fala do Papai Noel. Portanto, esta será exibida na tela `N` vezes: "**Ho Ho.. Ho!**". Lembrando que o último "Ho" deve ser seguido de um ponto de exclamação ("!"), resultando em "Ho!".  
  
Para isso iniciamos declarando as variáveis que serão utilizadas:
```

```c
        int N, i;
```

> `N` o número de repetições que o usuário deseja e `i` variável padrão utilizada no controle do `for`.

Reviewer: `N` representa o número de "Ho" que foi sorteado

Author: Não entendi o sentido de sorteado, N é apenas a quantidade de repetições da palavra "Ho".

> Fazemos a leitura do `N` no `scanf`:

Reviewer: ```suggestion
Fazemos, então, a leitura de `N` através da função `scanf`:
```

```c
        scanf("%d", &N);
```

Seguimos para a construção do `for` onde será feito os printes dos "Ho":

```c
        for (i = 1; i <= N; i++)
        {
                if(i==N)
                        printf("Ho!\n");
                else
                        printf("Ho ");
        }
```

> Temos que lembrar que o último "Ho" tem a forma "Ho!" e para fazer que essa regra seja cumprida usamos um `if`, no qual a regra a ser contemplada é a do `i` nossa variável iterativa, a qual controla nosso `for` e se ela for igual ao valor de `N` significara que o final da iteração chegou.

Reviewer: * Os textos que estão antes e logo após o código podem ser resumidos em um só para facilitar a linha de raciocínio e não precisar repetir as mesmas informações.
* O último parágrafo está bem confuso.. talvez repartir a explicação em mais frases possa ajudar.

> 

Reviewer: Acho legal colocar pelo menos um "Para aprender um pouco mais sobre" o `for` pelo menos (que é a parte mais complicadinha desse exercício)
